### PR TITLE
Cuda 11.5 is not supported and this should be documented

### PR DESCRIPTION
**Pull request by surak**
https://github.com/pytorch/pytorch/issues/70931

Telling people why things don't work save dozens of hours of time of the others.

### CODE REVIEW

**Comment by surak** [comment]
Signed the CLA

File: README.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ You will get a high-quality BLAS library (MKL) and you get controlled dependency
 Once you have [Anaconda](https://www.anaconda.com/distribution/#download-section) installed, here are the instructions.
 
 If you want to compile with CUDA support, install
-- [NVIDIA CUDA](https://developer.nvidia.com/cuda-downloads) 10.2 or above
+- [NVIDIA CUDA](https://developer.nvidia.com/cuda-downloads) 10.2 or above - Currently incompatible with 11.5
```
Comment (ngimel) [inline README.md:176]: this is not correct, master supports 11.5

Comment (surak) [reply README.md:176]: And what is the branch I can edit this for 1.10.2? One can't edit in tags

Comment (ngimel) [reply README.md:176]: cc @b0nol, I don't know what's the policy towards release branches.

Comment (seemethere) [reply README.md:176]: We can add it to the release branch, `release/1.10`

Although the central authority for what we do support can be found at https://pytorch.org/get-started/locally/

Comment (seemethere) [reply README.md:176]: master does support CUDA 11.5 though so this PR isn't actually necessary

**Comment by surak** [comment]
It’s obsolete as of 1.11

**Pull request closed without merging**